### PR TITLE
[Assignment1 code MineBot]

### PR DESCRIPTION
**Pull request by GrayFlash**

### CODE REVIEW

File: Submissions/Gaurav-ECE-SVNIT/Assignment1.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+        return sha256((str(self.timestamp)+str(self.data)+str(self.previousHash)).encode()).hexdigest()
```
Comment (johnsoncarl) [inline Submissions/Gaurav-ECE-SVNIT/Assignment1.py]: will work..!!

Just to make sure I checked it in Python editor, I found inconsistent use of tabs. please correct it in this line

**Review by johnsoncarl — CHANGES_REQUESTED** [review]
Please make the suggested changes and submit.

**Comment by Gaurav-ECE-SVNIT** [comment]
I have done the changes please merge it now.

**Comment by GrayFlash** [comment]
I have made the changes.

**Review by johnsoncarl — APPROVED** [review]
It works,,, good work 👍

**Pull request merged**